report-api/report-templates/payment_receipt.html

```diff
@@ -432,10 +450,10 @@
         <td colspan="2" class="receipt-footer">
           <div class="footer-info">
             {% if filingIdentifier %}
-              <span>Filing #{{filingIdentifier}}</span>
+            <span>Filing #{{filingIdentifier}}</span>
             {% endif %}
             {% if invoice['businessIdentifier'] %}
-              <span>Incorporation #{{invoice['businessIdentifier']}}</span>
+            <span>Incorporation #{{invoice['businessIdentifier']}}</span>
```

Comment:
Since we are popping business identifier from the response, probably we will need a None safe get here ?

Reply:
It seems the template if statement already handle the None type. Do you mean we should handle "businessIdentifier": "None" in payload?

Reply:
What I meant was;
`if invoice['businessIdentifier']` will throw error if the key `businessIdentifier` is not present in invoice dict.
Changing it to;
`if invoice.get('businessIdentifier', None)` will make it as None safe key.

Reply:
Fixed.